### PR TITLE
[EDU-5814] refactor: update links to videos about Data Stream integrations

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-5814]

### Changes

- Update links to videos about Data Stream integrations: Azure Monitor, Edge Storage

### Additional links

n/a.

[EDU-5814]: https://aziontech.atlassian.net/browse/EDU-5814?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by guiafonso-ol — APPROVED** [review]

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-blob.mdx [hunk 1]
```diff
@@ -75,6 +75,12 @@ After saving the configurations, your data will be streamed to the newly configu
 
 You can keep track of the calls made by Data Stream to Azure Blob on [Real-Time Events](/en/documentation/products/observe/real-time-events/). To do so, select **Data Source** > **Data Stream** and choose the filters options as you wish.
 
+Learn how to use Azure Blob Storage to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/KcufBkalznc?si=2AWrs2l745PpjHCb" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-blob.mdx:81]: title="How to use Azure Blob Storage to receive data from Data Stream"

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-monitor.mdx [hunk 2]
```diff
@@ -79,12 +79,14 @@ After saving the configurations, your data will be streamed to the newly configu
 
 You can keep track of the calls made by Data Stream to Azure Monitor on [Real-Time Events](/en/documentation/products/observe/real-time-events/). To do so, select **Data Source** > **Data Stream** and choose the filters options as you wish.
 
+Learn how to use Azure Monitor to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/P9kzC1Jmit0?si=Ai3HEVqwTGOPXgIO" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-monitor.mdx:85]: title="How to use Azure Monitor to receive data from Data Stream"

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-big-query.mdx [hunk 3]
```diff
@@ -178,6 +178,12 @@ After saving the configurations, your data will be streamed to the newly configu
 
 You can keep track of the calls made by Data Stream to Google BigQuery on [Real-Time Events](/en/documentation/products/observe/real-time-events/). To do so, select **Data Source** > **Data Stream** and choose the filters options as you wish.
 
+Learn how to use Google BigQuery to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/DUcfEBcY6xY?si=MjlpIJzHhG9ANwvt" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-big-query.mdx:184]: title="How to use Google BigQuery to receive data from Data Stream"

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-datadog.mdx [hunk 4]
```diff
@@ -82,5 +82,12 @@ You can keep track of the calls made by Data Stream to Datadog on [Real-Time Eve
 [Datadog](https://docs.datadoghq.com/) is a registered trademark of Datadog, Inc in the United States.
 
 
+Learn how to use Datadog to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/nqKMWorVX6s?si=BajW-IOsP8UBwVV2" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-datadog.mdx:88]: title="How to use Datadog to receive data from Data Stream"

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-elasticsearch.mdx [hunk 5]
```diff
@@ -88,12 +88,14 @@ After saving the configurations, your data will be streamed to the newly configu
 
 You can keep track of the calls made by Data Stream to Elasticsearch on [Real-Time Events](/en/documentation/products/observe/real-time-events/). To do so, select **Data Source** > **Data Stream** and choose the filters options as you wish.
 
+Learn how to use Elasticsearch to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/VdoISBuwYxI?si=r8sE599SVJbn8Xto" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-elasticsearch.mdx:94]: title="How to use Elasticsearch to receive data from Data Stream."

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-splunk.mdx [hunk 6]
```diff
@@ -84,6 +84,12 @@ After saving the configurations, your data will be streamed to the newly configu
 
 You can keep track of the calls made by Data Stream to Splunk on [Real-Time Events](/en/documentation/products/observe/real-time-events/). To do so, select **Data Source** > **Data Stream** and choose the filters options as you wish.
 
+Learn how to use  Splunk to receive data from Data Stream. Watch the video below:
+
+<div class="cms-embed">
+  <iframe width="600" height="400" src="https://www.youtube.com/embed/VT0JYYG7Rcw?si=Kqt5iec_p58PJvv-" loading="lazy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-splunk.mdx:90]: title="How to use Splunk to receive data from Data Stream"

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-blob.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-blob.mdx:81]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/KcufBkalznc?si=2AWrs2l745PpjHCb" loading="lazy" title="How to use Azure Blob Storage to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-monitor.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-azure-monitor.mdx:85]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/P9kzC1Jmit0?si=Ai3HEVqwTGOPXgIO" loading="lazy" title="How to use Azure Monitor to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-big-query.mdx (hunk 3 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-big-query.mdx:184]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/DUcfEBcY6xY?si=MjlpIJzHhG9ANwvt" loading="lazy" title="How to use Google BigQuery to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-datadog.mdx (hunk 4 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-datadog.mdx:88]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/nqKMWorVX6s?si=BajW-IOsP8UBwVV2" loading="lazy" title="How to use Datadog to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-elasticsearch.mdx (hunk 5 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-elasticsearch.mdx:94]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/VdoISBuwYxI?si=r8sE599SVJbn8Xto" loading="lazy" title="How to use Elasticsearch to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-splunk.mdx (hunk 6 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-splunk.mdx:90]:
```suggestion
  <iframe width="600" height="400" src="https://www.youtube.com/embed/VT0JYYG7Rcw?si=Kqt5iec_p58PJvv-" loading="lazy" title="How to use Splunk to receive data from Data Stream" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

**Review by evandroazion — APPROVED** [review]

**Pull request merged**